dist/api/bundler.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+var api = require('jspm/api');
+var whacko = require('whacko');
+var glob = require('glob');
+var fs = require('fs');
+var url = require('url');
+var path = require('path');
+
+var pluginName = 'view';
+var loader = api.Builder().loader;
```

Comment:
I believe the issue is that I am requiring the file as the API.prototype. This file should never be required if it is not needed. I will refector.